### PR TITLE
Add API for get-bot-info

### PR DESCRIPTION
**Pull request by hinoguma**
Update Messaging API October 2020
[https://developers.line.biz/en/news/2020/10/06/messaging-api-update-october-2020/](https://developers.line.biz/en/news/2020/10/06/messaging-api-update-october-2020/)

- [x] [Get bot info](https://developers.line.biz/en/reference/messaging-api/#get-bot-info)
- [ ]  Get webhook endpoint information
- [ ]  Set webhook endpoint URL
- [ ]  Test webhook endpoint

### CODE REVIEW

File: linebot/response.go [hunk 1]
```diff
@@ -93,6 +93,16 @@ type MessageConsumptionResponse struct {
 	TotalUsage int64
 }
 
+// BotInfoResponse type
+type BotInfoResponse struct {
+	UserID         string         `json:"userId"`
+	BasicID        string         `json:"basicId"`
+	PremiumID      string         `json:"premiumId"`
```
Comment (kkdai) [inline linebot/response.go:100]: Do you miss the "displayName"? 

Refer https://developers.line.biz/en/reference/messaging-api/#get-bot-info-response

File: linebot/get_bot_info_test.go [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+			ResponseCode: 200,
+			Response:     []byte(`{"userId":"u206d25c2ea6bd87c17655609a1c37cb8","basicId":"@012abcde","premiumId":"PremiumId","pictureUrl":"https://example.com/abcdefghijklmn","chatMode":"chat","markAsReadMode":"manual"}`),
+			Want: want{
+				URLPath:     APIEndpointGetBotInfo,
+				RequestBody: []byte(""),
+				Response: &BotInfoResponse{
+					UserID:         "u206d25c2ea6bd87c17655609a1c37cb8",
+					BasicID:        "@012abcde",
+					PremiumID:      "PremiumId",
+					PictureURL:     "https://example.com/abcdefghijklmn",
```
Comment (kkdai) [inline linebot/get_bot_info_test.go:52]: Also for "displayName"

Refer https://developers.line.biz/en/reference/messaging-api/#get-bot-info-response

**Review by kkdai — CHANGES_REQUESTED** [review]
Thank you for your PR, I have reviewed your PR.
Could you help to check for "displayName"

**Comment by hinoguma** [comment]
Thank you for review.


> Thank you for your PR, I have reviewed your PR.
> Could you help to check for "displayName"

I fixed missing displayName.
Could you please review again ?

File: linebot/get_bot_info_test.go [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Millisecond)
+	defer cancel()
+	_, err = client.GetBotInfo().WithContext(ctx).Do()
+	expectCtxDeadlineExceed(ctx, err, t)
+}
+
+func BenchmarkGetBotInfo(b *testing.B) {
+	server := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		defer r.Body.Close()
+		w.Write([]byte(`{"userId":"U","basicId":"@B","premiumId":"P","pictureUrl":"https://","chatMode":"chat","markAsReadMode":"manual"}`))
```
Comment (kkdai) [inline linebot/get_bot_info_test.go:189]: here also need "displayName" as well.

**Review by kkdai — CHANGES_REQUESTED** [review]
Thank you, you almost finish with this PR.
Please help to fix the final part.

**Comment by hinoguma** [comment]
Oh, sorry... I fixed and please check it.

**Comment by kkdai** [comment]
Thank you for your hard work, it looks good to me. 
Looking forward to seeing you for the rest of the API.

**Review by kkdai — APPROVED** [review]
LGTM

**Pull request merged**